test/lint.js

```diff
@@ -159,6 +160,7 @@ var hasErrors = argv.files
     );
 hasErrors = testCompareFeatures() || hasErrors;
 hasErrors = testMigrations() || hasErrors;
+hasErrors = testStyling() || hasErrors;
```

Comment:
It doesn't look like `testStyling` (or the future's `testFormat`) returns anything. It looks as if this can only ever fail.

Reply:
Ah, good catch!  Preferably, I'd love to get #5300 merged (which is a way this issue could get fixed, since it bases "hasErrors" off of whether we've gotten a `console.error`), but I'll see what I can do for this.  :wink:

Reply:
Alright, got this fixed up, along with another potential issue regarding the asynchronous nature of `exect()`.  This is ready for re-review!